app/steps/ui/payment/status/template.html

```diff
@@ -8,7 +8,7 @@ <h1 class="govuk-heading-l">{{ content.question }}</h1>
 
     {% if errors == undefined %}
         {% if fields.paymentDue.value == "true" %}
-            <p class="govuk-body">{{ content.paragraph | safe }}</p>
+            <p class="govuk-body">{{ content.paragraph1 | safe }}</p>
```

Comment:
Not an optimisation

Reply:
It was also changed it in the translation file, won't affect anything, leave this.